### PR TITLE
Add "lightpath" package

### PR DESCRIPTION
**Pull request by tangkong**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

cc @klauer, @ZLLentz

### CODE REVIEW

File: recipes/lightpath/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - pip check
+    - lightpath --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/pcdshub/lightpath
+  doc_url: https://pcdshub.github.io/lightpath
+  dev_url: https://github.com/pcdshub/lightpath
+  license: SLAC Open
```
Comment (klauer) [inline recipes/lightpath/meta.yaml:50]:
```suggestion
  license: LicenseRef-BSD-3-Clause-SLAC
```

File: recipes/lightpath/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  license_family: Other
```
Comment (klauer) [inline recipes/lightpath/meta.yaml:51]:
```suggestion
  license_family: BSD
```

File: recipes/lightpath/LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+Copyright (c) 2018, The Board of Trustees of the Leland Stanford Junior
```
Comment (klauer) [inline recipes/lightpath/LICENSE.md:1]: The `LICENSE.md` is included in the repository (https://github.com/pcdshub/lightpath/blob/master/LICENSE.md) so it doesn't need to be included here.

**Review by klauer — CHANGES_REQUESTED** [review]

**Comment by klauer** [comment]
I am willing to be a maintainer of this recipe.

**Comment by tangkong** [comment]
@conda-forge/help-python, ready for review!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**